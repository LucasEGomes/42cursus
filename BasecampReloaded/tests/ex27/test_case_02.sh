NO_EXT="${0%.*}"
OUTPUT=${NO_EXT}.output
RESULT=${NO_EXT}.user
EXPECTED=${NO_EXT}.expected
mkdir $NO_EXT/
cp -r $ROOT_DIR/ex27/* $NO_EXT/
echo -n > $RESULT
echo -n > $OUTPUT
find $NO_EXT/ -type f -name ft_display_file | sort >> $RESULT 2>> $OUTPUT
make all -C $NO_EXT/ 2>> $OUTPUT
find $NO_EXT/ -type f -name ft_display_file | sort >> $RESULT 2>> $OUTPUT
$NO_EXT/ft_display_file "Hello" "There" >> $RESULT 2>> $OUTPUT
$NO_EXT/ft_display_file "Hello" "There" "General" "Kenobi" >> $RESULT 2>> $OUTPUT
make clean -C $NO_EXT/ 2>> $OUTPUT
find $NO_EXT/ -type f -name *.o | sort >> $RESULT 2>> $OUTPUT
make fclean -C $NO_EXT/ 2>> $OUTPUT
find $NO_EXT/ -type f -name ft_display_file | sort >> $RESULT 2>> $OUTPUT
make all -C $NO_EXT/ 2>> $OUTPUT
make fclean -C $NO_EXT/ 2>> $OUTPUT
find $NO_EXT/ -type f -name ft_display_file | sort >> $RESULT 2>> $OUTPUT
bash assert.sh $NO_EXT $OUTPUT $RESULT $EXPECTED
bash tear_down.sh $NO_EXT
