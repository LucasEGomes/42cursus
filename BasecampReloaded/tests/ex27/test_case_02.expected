test_case_02/ft_display_file
Too many arguments.
Too many arguments.
